test/contracts/libraries/trie/Lib_SecureMerkleTrie.spec.ts

```diff
@@ -7,7 +7,7 @@ import { Contract } from 'ethers'
 /* Internal Imports */
 import { TrieTestGenerator } from '../../../helpers'
 
-const NODE_COUNTS = [1, 2, 128, 256, 512, 1024, 2048, 4096]
+const NODE_COUNTS = [1, 2, 128]
```

Comment:
Just set this to speed up our test suite

Reply:
Maybe too aggressive in removing test coverage? I'm not super familiar with the implementation so it would be best to get an ok from the people who wrote the code